src/main/java/com/google/cloud/teleport/templates/TextToBigQueryStreaming.java

```diff
@@ -268,9 +272,9 @@ public static PipelineResult run(TextToBigQueryStreamingOptions options) {
     FailsafeElement<String, String> failsafeElement;
     try {
 
-      failsafeElement =
-          FailsafeElement.of(
-              insertError.getRow().toPrettyString(), insertError.getRow().toPrettyString());
+      TableRow row = insertError.getRow();
```

Comment:
This block can then use the JsonFactory provided by Transport:

   JsonFactory factory = Transport.getJsonFactory();
      failsafeElement =
          FailsafeElement.of(
              factory.toPrettyString(insertError.getRow()),
              factory.toPrettyString(insertError.getRow()));
      failsafeElement.setErrorMessage(factory.toPrettyString(insertError.getError()));